### DISCUSSION TITLE
Command Border

### DISCUSSION THREAD

**Discussion by deanle8** [BG-Software-LLC/SuperiorSkyblock2#1034] [Implemented Feature Requests] [answered]
Add command to change border color like:
`/is border red`

**Comment by YouSeeMeRunning2** [comment]
This is already a feature. 
/is border

**Reply by deanle8** [reply]
I want to use command to change instead of using gui :(

**Comment by OmerBenGera** [comment, marked as answer]
Implemented :)
Will be added to the official builds soon
